cmd/mesh/upgrade.go

```diff
@@ -39,6 +39,14 @@ const (
 	// The maximum number of attempts that the command will check for the upgrade completion,
 	// which means only the target version exist and the old version pods have been terminated.
 	upgradeWaitCheckVerMaxAttempts = 60
+
+	// This message provide the guide of how to upgrade Istio data plane
+	upgradeSidecarMessage = "To upgrade the Istio data plane, you will need to re-inject it.\n" +
+		"If you’re using automatic sidecar injection, you can upgrade the sidecar by doing a rolling" +
+		" update for all the pods:\n" +
+		"    kubectl rollout restart deployment --namespace <namespace with auto injection>\n" +
```

Comment:
`kubectl rollout restart deployment` does seem to exist for me, and I have the latest kubectl that comes with `gcloud components install kubectl`. What am I doing wrong?

Reply:
Ok, if I download the latest non-gcloud, it has the command so it must be very new. It might confused some users if many still have older versions.